tasks/message/message.go

```diff
@@ -3,6 +3,7 @@ package message
 import (
```

Comment:
I see what happened, this is duplicated in #166. This looks fairly mechanical, I'd be fine closing this and landing everything in #166 if that's easier for you -- will leave up to you.